docs/analysis-techniques.md

```diff
@@ -221,61 +221,61 @@ flowchart TB
+Static analysis can also be used to help reduce the static-analysis state-space-explosion problem. This is because static analysis can be used to get a better picture of the whole code with all its dependencies. Meaning that branches that are not interacting with the state variables can be pruned earlier for example. For Solidity specifically, this was presented in the [paper for MPro](https://arxiv.org/pdf/1911.00570.pdf). MPro improved the symbolic execution tool for Solidity called [Mythril](https://github.com/ConsenSys/mythril/tree/develop) with the static analysis tool for Solidity called [Slither](https://github.com/crytic/slither).
 
 ## 8 A note of formal verification
 
-The techniques discussed so far will only tell you that the assertions (requirements) that you want to verify in your program are met within the configuration given to the tools. Meaning that if an assertion could be violated after `n` amount of steps, but to reduce the state-space the configuration only allowed `n-1` steps, that the violation might not be found (and this not reported).
+The techniques discussed so far will only tell you that the assertions (requirements) that you want to verify in your program are met within the configuration given to the tools. Meaning that if an assertion could be violated after `n` amount of steps, but to reduce the state-space the configuration is only allowed `n-1` steps, the violation might not be found (and thus not reported).
 
-Also, the tools can only verify what is given to them, so if the assertion does not cover the associated required in a function design, then the assertion might not be violated while the assertion itself is not correct. Giving a false impression of success.
+Also, the tools can only verify what is given to them, so if the assertion does not cover the associated requirement in a function design, then the assertion might not be violated while the assertion itself is not correct. Giving a false impression of success.
 
 ---
 
 ## 9 How Byont uses these techniques
 
-At Byont we make heavy use of Foundry's fuzzing capabilities. Besides that, we are also making use of Slither, solc's SMTChecker, Mythril and KEVM. We are currently in the process of optimizing the process of incorporating these tools in our development flow.
+At Byont we make heavy use of Foundry's fuzzing capabilities. Besides that, we are also making use of Slither, solc's SMTChecker, Mythril, and KEVM. We are currently in the process of optimizing the process of incorporating these tools in our development flow.
```

Comment:
Might be nice to explain a bit more about what KEVM is / does or provide a link to a resource that does so

Reply:
[da3b96a](https://github.com/Byont-Ventures/smart-contract-analysis-tools/pull/47/commits/da3b96aa5950dc586ca73112f0d73e6b6a8fb1c4): I've update this part a little bit by adding links to the docs and git repo's of the different tools. I think that this page should not go into much more details about the tools themselves. Both because 1) it can become quite long what going into the differences of each tools, and 2) because in a next blog/documentation page, the different tools will be demonstrated by means of a demo.